docs/README.md

```diff
@@ -30,3 +30,9 @@ Install our Github App to autopropagate changes from youre repo to your deployme
 
 - Mintlify dev isn't running - Run `mintlify install` it'll re-install dependencies.
 - Page loads as a 404 - Make sure you are running in a folder with `mint.json`
+
+### Swagger
+
+- For developers modifying world-engine: world-engine is compatible with swag. Run `make swagger` to regenerate `swagger.json`, `swagger.yaml`, `swagger.yml` and `docs.go` in the docs folder. 
```

Comment:
```suggestion
- For developers modifying world-engine: world-engine is compatible with swag. Run `make swagger` to regenerate `swagger.json`, `swagger.yml` and `docs.go` in the docs folder. 
```

Reply:
This is outdated. I'm going to change it.